tests/contrib/test_pydantic.py

```diff
@@ -0,0 +1,246 @@
+import dataclasses
+import uuid
+from datetime import datetime, timedelta
+from ipaddress import IPv4Address
+from typing import Annotated, Any, List, Sequence, Tuple, TypeVar
+
+from annotated_types import Len
+from pydantic import BaseModel, Field, WithJsonSchema
+
+from temporalio import activity, workflow
+from temporalio.client import Client
+from temporalio.contrib.pydantic import pydantic_data_converter
+from temporalio.worker import Worker
+
+SequenceType = TypeVar("SequenceType", bound=Sequence[Any])
+ShortSequence = Annotated[SequenceType, Len(max_length=2)]
+
+
+class MyPydanticModel(BaseModel):
```

Comment:
I will say for many users they will be importing this model from a different module on passthrough. This test confirms the model works when defined _in_ the sandbox, but I wonder if we also need at least a little coverage for models defined out of the sandbox (and passed through).

Reply:
I've structured the test suite so that `models.py` (containing several model definitions) is imported in-sandbox, but a few of the models are defined in a different file and imported under the `unsafe.imports_passed_through` context manager.